index.html

```diff
@@ -157,87 +157,87 @@ <h2>Stages</h2>
 
 <h2>In cases where the committee does not come to consensus</h2>
 
-<p>The committee may come to a point where consensus is not reached in committee regarding the feature. In this case, the committee must record a good description of why a proposal did not advance. This should be done both in the meeting notes and within an issue in the proposal's tracker, but not limited to those. This allows us to understand issues in the proposal and similar proposals in a coherent way.
+<p>The committee may come to a point where consensus is not reached in committee regarding the feature. In this case, the committee must record a good description of why a proposal did not advance. This should be documented in both the meeting notes and an issue in the proposal's repository, but not limited to these. This helps us understand issues in the proposal and similar proposals in a cohesive way.
 
-<p>There are two forms that this sometimes takes, the first is the violation of a constraint and the other is colloquially known as a block. Other forms exist but are not discussed directly here.
+<p>There are two forms that this sometimes takes, the first is the violation of a constraint and the other is colloquially known as a "block". Other forms exist but are not addressed here.
 
 <p>In the first case, delegates may consider that the violation of a constraint is sufficiently serious reason to withhold their consensus for stage advancement. The dissenting delegate(s) and the champion(s) should work together accordingly to resolve the issue.
 
 <p>Not all issues with proposals are easily solvable. Some issues are too fundamental and serious, requiring a significant rework of the proposal, or may be unsolvable. In these situations, if consensus is withheld, it might be referred to colloquially as a "block". The proposal will require substantial work to address the concern, may need to be rethought all together, or may not have enough justification to pursue at this time.
 
-<p>When possible, it is preferable to raise an actionable constraint. The committee does not have an established concept of a rejected proposal--it is always possible for the champion to make changes and come back to ask for consensus.
+<p>When possible, it is preferable to raise an actionable constraint. The committee does not have an established concept of a rejected proposal -- it is always possible for the champion to make changes and come back to ask for consensus.
```

Comment:
`&mdash;`?

Reply:
iirc `--` and `&mdash;` are the same thing in markdown

Reply:
1. this isn't markdown
2. there's no need to use entity references when we have glorious glorious UNICODE
3. nonetheless, I prefer using characters available on a standard `en` keyboard. :)

Reply:
I just thought it would be better to prioritize how it displays, but I don't feel strongly about it.

Reply:
I'm actually undoing this change.  APA and CMOS both say no spaces.  😳